styles.css

```diff
@@ -165,6 +165,7 @@ form , .timer-view {
 }
 
 .error-message {
+  display: hidden;
```

Comment:
I don't think that this should be here, we use the hidden class to define whether the element should be hidden.